### PR TITLE
Owl Carousel

### PR DESCRIPTION
**Pull request by Mahak008**
Here is the screenshot of the feature added.
Screenshot (331)

### CODE REVIEW

**Comment by Susaksham** [comment]
> Here is the screenshot of the feature added. Screenshot (331)

Please add the readme and comments in all the files , There are also some files that are not rendering so check  the [template guidelines](https://github.com/mudit023/templates/blob/main/TEMPLATE_GUIDELINES.md)

**Comment by Susaksham** [comment]
Please add the readme , gif and comments in all the files , check the [template guidelines](https://github.com/mudit023/templates/blob/main/TEMPLATE_GUIDELINES.md)

**Comment by Susaksham** [comment]
> Here is the screenshot of the feature added. Screenshot (331)

Please add the readme , gif and comments in all the files , check the [template guidelines](https://github.com/mudit023/templates/blob/main/TEMPLATE_GUIDELINES.md)

**Comment by Mahak008** [comment]
Ok @Susaksham

**Comment by Mahak008** [comment]
I am sorry, I closed this request by mistake. Kindly reconsider it once @mudit023

File: Owl Carousel/Readme.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Owl Carousel
+
```
Comment (mudit023) [inline Owl Carousel/Readme.md:2]: Please add a proper readme file like this: [example](https://github.com/mudit023/templates/blob/main/animated-newsletter/Readme.md)

File: Owl Carousel/Readme.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+This code can be used to create moving cards like profile cards, blogs etc.
```
Comment (mudit023) [inline Owl Carousel/Readme.md:3]: Add a gif of your template that shows how your template looks and behaves.

File: Owl Carousel/index.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Owl Carousel</title>
+        <link rel="stylesheet" href="styles.css">
+        <link rel="stylesheet" href="owl.carousel.min.css">
+    </head>
+
+    <body>
+        <div class="slider">
```
Comment (mudit023) [inline Owl Carousel/index.html:13]: Add proper comments where ever needed. To make it easier for the user to use your template.

File: Owl Carousel/styles.css [hunk 4]
```diff
@@ -0,0 +1,71 @@
+.slider {
+    margin-top: 5%;
```
Comment (mudit023) [inline Owl Carousel/styles.css:2]: make use of CSS variables.

**Review by mudit023 — CHANGES_REQUESTED** [review]
Thanks for the contribution. Please follow our [template guidelines](https://github.com/mudit023/templates/blob/main/TEMPLATE_GUIDELINES.md) and we'll merge your PR.

**Comment by Mahak008** [comment]
> Thanks for the contribution. Please follow our [template guidelines](https://github.com/mudit023/templates/blob/main/TEMPLATE_GUIDELINES.md) and we'll merge your PR.

Ok, I will made all the required changes. Thanks for your response.

**Comment by Mahak008** [comment]
@mudit023 @Susaksham Can you please help me out how can I make changes to the readme.md file. Since, I accidentally deleted the forked repo due to which I am unable to make changes to Readme.md .

**Comment by Susaksham** [comment]
> @mudit023 @Susaksham Can you please help me out how can I make changes to the readme.md file. Since, I accidentally deleted the forked repo due to which I am unable to make changes to Readme.md .
@Mahak008 
if you are talking about upstreaming that is how you can do it 
image

**Comment by mudit023** [comment]
@Mahak008 let me know if it's okay for you.

**Comment by Mahak008** [comment]
> @Mahak008 let me know if it's okay for you.

Yeah sure

**Comment by mudit023** [comment]
> > @Mahak008 let me know if it's okay for you.
> 
> Yeah sure

Okay. I'm closing this PR.

**Pull request closed without merging**